src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public BalanceOperation(FruitTransactionDao fruitTransactionDao) {
+        this.fruitTransactionDao = fruitTransactionDao;
+    }
+
+    @Override
+    public void recount(FruitTransaction transaction) {
```

Comment:
The `recount` method name is not very descriptive of its purpose. Consider renaming the method to more accurately reflect that it's handling the balance operation, such as `applyBalance` or `updateBalance`.